forge-gui/res/cardsfolder/e/elderscale_wurm.txt

```diff
@@ -9,4 +9,6 @@ R:Event$ LifeReduced | ActiveZones$ Battlefield | ValidPlayer$ You | Result$ LT7
 SVar:ReduceLoss:DB$ ReplaceEffect | VarName$ Result | VarValue$ X
 SVar:X:ReplaceCount$Result/NMinus.7
 SVar:ElderscaleCondition:Count$YourLifeTotal
+#NOTE: If the opponent controls the wurm, AI won't take into consideration that combat damage may be ineffective.
```

Comment:
AI:RemoveDeck:All flag is only for the AI's own decks, not when the opponent has one in his deck. This won't stop human players from having it and abusing it vs. the AI. Thus, the flag isn't the best here. If there's a concern that the AI misplays it or plays it at the wrong time, then other hints may be added as needed.

Reply:
Fair enough.